app/views/search/index.html.erb

```diff
@@ -110,6 +110,8 @@
           <p class="collection-filters">
             <input type="checkbox" id='has-granules' data-bind="checked: query.hasGranules"/>
             <label for="has-granules">Include collections without granules</label>
```

Comment:
It will require an update on all the collection fixtures, which is why we gave this ticket a 3 points. (@john, you can manually edit the collection fixtures if it is easier than delete-all and re-gen them.)